docs/advanced_usage.md

```diff
@@ -0,0 +1,232 @@
+Started](getting_started.md) docs or run `notary --help` before continuing. Once
+you understand what these flags do, you must provide your own values for these
+options while following this document. You can also configure these options, see
+[advanced configuration options](reference/index.md) for more information.
+
+## Initialize a Repository</a>
+
+Before adding and signing content to a repository, you must first initialize that repository.
+
+```
+$ notary init example.com/collection
+
+No root keys found. Generating a new root key...
+You are about to create a new root signing key passphrase. This passphrase
+will be used to protect the most sensitive key in your signing system. Please
+choose a long, complex passphrase and be careful to keep the password and the
+key file itself secure and backed up. It is highly recommended that you use a
+password manager to generate the passphrase and keep it safe. There will be no
+way to recover this key. You can find the key in your config directory.
+Enter passphrase for new root key with ID 1f54328:
+Repeat passphrase for new root key with ID 1f54328:
+Enter passphrase for new targets key with ID 1df39fc (example.com/collection):
+Repeat passphrase for new targets key with ID 1df39fc (example.com/collection):
+```
+
+Initializing a repository will generate the following items; all keys use
+asymmetric algorithms, but there is no requirement that they all use the _same_
+algorithm:
+
+- If no root key is found, an initial root key will be generated. This key will be used as the default root of trust for all your repositories.
+- A targets key and a snapshot key. The same password is used to encrypt both of these as the security profile of them (when both held by the author of the repository) is identical. This is why you will not be asked for a snapshot key password.
+- A timestamp key. This is generated by the server on a request from the client, returning just the public key. The server holds the private key and will sign timestamps on behalf of the user.
+- Stub signed notary metadata. This stages the base version of the trust metadata for the repository. It will be finalized when it is published to the server.
+
+## Add and remove Targets
+
+It's simple to add targets to a repository with notary CLI:
+```
+$ notary add example.com/collection v1 my_file.txt
+```
+
+The above command adds the local file `my_file.txt` (this file must exist relative to the current working directory) under the target name `v1` to the `example.com/collection` repository we set up. The contents of the local file are not actually added to the repository - a "target" consists of the
+file path and one or more checksums of the contents.
+
+Note that this is an offline command, and we must run a `notary publish example.com/collection` for the add to take effect.
+
+To remove targets, we use the `notary remove` command, specifying the GUN and target name.
+```
+$ notary add example.com/collection v1
```

Comment:
we should change `add` to `remove` for this command -- apologies, seems like this might've been my typo from the original PR.